### ISSUE TITLE
Postvaccination yellow fever

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7617]
**Preferred term label:**
Postvaccination yellow fever

**Synonyms**

**Definition (free text, please give PubMed ID)**
Infection with the yellow fever virus of the live-attenuated vaccine.

**Parent term (use hpo.jax.org/app)**
Infection following live vaccination HP:0020085

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 619935 

PMID: 31270247
1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
